### ISSUE TITLE
Create a blog post about on use of metadata (vs layers) in data catalog and with Kedro-Viz.

### ISSUE THREAD

**Issue by NeroOkwa** [kedro-org/kedro-devrel#93]
[Placeholder]
This blog post covers the introduction of the metadata attribute, compared to 'layer' which was previously defined in the data catalog. 

This post should explain why we made the change, and how users of 'layer' can use the 'metadata' attribute.

More details: [#1076](https://github.com/kedro-org/kedro/issues/1076), [#2439](https://github.com/kedro-org/kedro/issues/2439)

**Comment by stichbury** [comment]
I don't think this is a complete blog post. If you can think of a new example of using layers and draft out a skeleton of the blog post (subheadings etc) to present that example, we can consider it as more compelling and complete it for publication. 

Please can you or Rashida come up with a full brief? The template is here: https://github.com/kedro-org/kedro-devrel/wiki/Kedro-blog-post-brief-template

**Comment by tynandebold** [comment]
Closing this, as we think the docs are sufficient to explain this work to the users.

**Comment by tynandebold** [comment]
Closing this, as we think the docs are sufficient to explain this work to the users.
